apiary.apib

```diff
@@ -276,7 +237,9 @@ This is the API documentation for the Blackboard Transact Slate project.  The in
 + PinChangeRequired: false (boolean) - Does the operator have to change their pin.
 + CardNumber: 600001 (string) - The operator`s card number.
 + IssueNumber: null (string) - The operator`s card issue number.
-+ Roles (array[string]) - The list of roles (identified as a list of guids corresponding to role ids) assigned to the operator.
```

Comment:
It seems the Apiary is having a trouble supporting default values for declared nested types (does not show the value at all). The only way is to omit the type in declaration and mention in at the value itself.